apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -5432,10 +5386,6 @@ suite('conversation.js >', function() {
 
         placeholder.dispatchEvent(new CustomEvent('input', { bubbles: true }));
       });
-
-      test('An `recipientschange` event is sent', function() {
-        sinon.assert.calledOnce(onRecipientsChange);
-      });
```

Comment:
shouldn't you replace all these assert by an assert that Compose.refresh is called ?